easybuild/easyconfigs/p/plinkQC/plinkQC-0.3.3-foss-2020a-R-4.0.0.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'Bundle'
+
+name = 'plinkQC'
+# Listed upstream as 0.3.3 unreleased
+version = '0.3.3'
+versionsuffix = '-R-%(rver)s'
+local_git_commit = 'b83c1ec'
+
+homepage = 'https://meyer-lab-cshl.github.io/plinkQC/'
+description = """plinkQC is a R/CRAN package for genotype quality control in genetic association studies.
+ It makes PLINK basic statistics (e.g.missing genotyping rates per individual, allele frequencies per genetic
+ marker) and relationship functions easily accessible from within R and allows for automatic evaluation of the
+ results."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+dependencies = [
+    ('R', '4.0.0'),
+]
+
+exts_default_options = {
+    'source_urls': [
+        'https://cran.r-project.org/src/contrib/Archive/%(name)s',  # package archive
+        'https://cran.r-project.org/src/contrib/',  # current version of packages
+        'https://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
+
+    ],
+    'source_tmpl': '%(name)s_%(version)s.tar.gz',
+}
+
+exts_defaultclass = 'RPackage'
+exts_filter = ("R -q --no-save", "library(%(ext_name)s)")
+
+exts_list = [
+    ('UpSetR', '1.4.0', {
+        'checksums': ['351e5fee64204cf77fd378cf2a2c0456cc19d4d98a2fd5f3dac74b69a505f100'],
+    }),
+    (name, version, {
+        'filename': 'meyer-lab-cshl-plinkQC-%(local_git_commit)s',
+        'source_tmpl': 'b83c1ec.tar.gz',
+        'source_urls': ['https://github.com/meyer-lab-cshl/plinkQC/archive/'],
+        'checksums': [None],
+    }),
+]
+
+unpack_options = '--strip-components=1'
```

Comment:
this line can be removed (I does not do anything for extensions)

Reply:
Also removed.